src/shogun/neuralnets/NeuralNetwork.cpp

```diff
@@ -813,9 +829,13 @@ void CNeuralNetwork::init()
 		"Parameters", MS_NOT_AVAILABLE);
 	SG_ADD(&m_param_regularizable, "param_regularizable",
 		"Parameter Regularizable", MS_NOT_AVAILABLE);
-	SG_ADD((CSGObject**)&m_layers, "layers",
-		"DynamicObjectArray of NeuralNetwork objects",
-		MS_NOT_AVAILABLE);
+	SG_ADD(
+	    &m_layers, "layers", "DynamicObjectArray of NeuralNetwork objects",
+	    MS_NOT_AVAILABLE);
+	SG_ADD(&m_auto_quick_initialize, "auto_quick_initialize", "auto_quick_initialize", MS_NOT_AVAILABLE);
```

Comment:
This needs a better description.
"Whether the network is automatically connected and initialized on training."